policyengine_us/variables/gov/states/al/tax/income/deductions/al_federal_income_tax_deduction.py

```diff
@@ -0,0 +1,14 @@
+from policyengine_us.model_api import *
+
+
+class al_federal_income_tax_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Alabama federal income tax deduction"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.revenue.alabama.gov/ultraviewer/viewer/basic_viewer/index.html?form=2023/01/22f40abk.pdf#page=20"
+    defined_for = StateCode.AL
+
+    adds = "gov.states.al.tax.income.deductions.federal_tax.countable_sources"
+    subtracts = ["al_federal_income_tax_deduction_credits"]
```

Comment:
you can skip the intermediate variable and use the parameter here like above

Reply:
I am not sure if the subtracts function works the same way, we tried it in a previous commit and it did not compute the list

Reply:
@nikhilwoodruff is this a bug in core?

Reply:
>Bump policyengine-core in setup.py to >=2.11.1
Could you please do this? CI tests will pass but local ones might not if users have an old version of core installed.

Reply:
Line 40 in the `setup.py`

![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/3309dac1-3ab9-4684-bfb7-4b13cc3e1874)